### PR TITLE
update searchResults page

### PR DESCRIPTION
**Pull request by rabe200**

### CODE REVIEW

File: components/SearchResults/index.js [hunk 1]
```diff
@@ -1,50 +1,79 @@
...
+  color: white;
+  list-style-type: none;
+  margin: 0;
+  padding-left: 36px;
+  padding-right: 36px;
+  font-size: 1.2em;
 `;
 
-const StyledList = styled.ul`
+const StyledList = styled.div`
```
Comment (PacoSchwab) [inline components/SearchResults/index.js:24]: All fine, just remember that naming a div as a "StyledList" doesn't follow 100% semantic HTML rules for SEO and accessibility ;) But it still is possible though.

File: components/SearchResults/index.js [hunk 2]
```diff
@@ -104,43 +137,132 @@ export default function SearchResults() {
...
+            <StyledBoldText>{cardByMeaningRev.length}</StyledBoldText>
+          </i>
+          <i>
+            Meaning upside:
+            <StyledBoldText>{cardByMeaningUp.length}</StyledBoldText>
+          </i>
+        </StyledSearchStats>
+        <select
+          onChange={(e) => renderResults(e)}
+          style={{
```
Comment (PacoSchwab) [inline components/SearchResults/index.js:159]: Maybe you could avoid using inline styling here?

**Review by PacoSchwab — APPROVED** [review]
LGTM, just tiny comments

File: components/SearchResults/index.js (hunk 2 above)
Comment (rabe200) [reply components/SearchResults/index.js:159]: tried it, but it won't apply : 
image

### i will implement this idea at a later point : [LINK](https://plainenglish.io/blog/custom-select-component-in-react-and-styled-components-e1f3b0a3a287)

**Pull request merged**